README.md

```diff
@@ -0,0 +1,95 @@
+[xarray](http://xarray.pydata.org/en/stable/). You can see real-world
+use cases on the [USAGE](https://github.com/GeoscienceAustralia/dea-notebooks/blob/stable/USAGE.rst)
+page.
+
+**Contribute** &mdash; DEA Notebooks is open-source and we would love your contribution!
+Learn [how to contribute](#contribute) below.
+
+**License** &mdash; The code in this repository is licensed under the [Apache
+License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0). DEA
+data is licensed under the [Creative Commons by Attribution 4.0
+license](https://creativecommons.org/licenses/by/4.0/).
+
+**Documentation** &mdash; See the [DEA Notebooks
+Wiki](https://github.com/GeoscienceAustralia/dea-notebooks/wiki).
+
+**Contact** &mdash; For assistance with any of these notebooks and tools,
+please ask a question on our [Open Data Cube Slack
+channel](http://slack.opendatacube.org/) or on the [GIS Stack
+Exchange](https://gis.stackexchange.com/questions/tagged/open-data-cube)
+using the `open-data-cube` tag. You can also [report an issue on this
+repository](https://github.com/GeoscienceAustralia/dea-notebooks/issues).
+
+**Citation** &mdash; If you use this repository in your work, please reference
+it with the following citation.
+
+> Krause, C., Dunn, B., Bishop-Taylor, R., Adams, C., Burton, C., Alger,
+> M., Chua, S., Phillips, C., Newey, V., Kouzoubov, K., Leith, A.,
+> Ayers, D., Hicks, A., DEA Notebooks contributors 2021. Digital Earth
+> Australia notebooks and tools repository. Geoscience Australia,
+> Canberra. <https://doi.org/10.26186/145234>
+
+## Browse the DEA Knowledge Hub
+
+The Knowledge Hub is where you can easily view the notebooks and read DEA Tools documentation. You can browse the following sections.
+
+* **[Beginner’s guide](https://docs.dea.ga.gov.au/notebooks/Beginners_guide/)** &mdash;
+  Introductory notebooks aimed at introducing Jupyter Notebooks and how to load,
+  plot, and interact with DEA data.
+* **[DEA products](https://docs.dea.ga.gov.au/notebooks/DEA_products/)** &mdash;
+  Notebooks introducing DEA's satellite datasets and derived products,
+  including how to load each dataset and any special features of the data.
+* **[How-to guides](https://docs.dea.ga.gov.au/notebooks/How_to_guides/)** &mdash;
+  A recipe book of simple code examples demonstrating how to perform common
+  geospatial analysis tasks using DEA and open-source software.
+* **[Interactive apps](https://docs.dea.ga.gov.au/notebooks/Interactive_apps/)** &mdash;
+  Interactive apps and widgets that require little or no coding to run.
+* **[Real-world examples](https://docs.dea.ga.gov.au/notebooks/Real_world_examples/)** &mdash;
+  More complex case study workflows demonstrating how DEA can be
+  used to address real-world problems.
+* **[DEA Tools Package](https://docs.dea.ga.gov.au/notebooks/Tools/)** &mdash;
+  An open-source Python package containing functions and
+  algorithms to assist in analysing DEA data.
```

Comment:
Can we add a link to the rendered DEA Tools API docs here?

Reply:
Ignore this, it's already linked!

Although, perhaps we should instead use relative links here so they open up correctly on the Sandbox? Could always leave the DEA Tools one as it is.